src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
Maybe. But we should be consistent here with other window globals, either we define modules for all of them or none. And since my 76df9e4e389d80bff410a9e5f08b848de1d21a2f was going in the "more var micro-modules" direction, I'd like to hear what @gibson042 & @timmywil have to say here.